Sprint 2

Prerendered opened the pull request:
# Features:
- Student Dashboard

EmmanuelBuruvuru reviewed (approved):
#18 

Pull Request approved

Prerendered commented:
#18 
# Features:
- Better student dashboard
- block/unblock videos after viewed
- communication with db for subjects/topics
- added quotes API : added limits to API
